Source/WTF/wtf/PlatformEnable.h

```diff
@@ -572,6 +572,9 @@
 #define ENABLE_CFPREFS_DIRECT_MODE 0
 #endif
 
+#if !defined(ENABLE_BACKDROP_FILTER)
+#define ENABLE_BACKDROP_FILTER 0
```

Comment:
Do we really need this? If we do, I'd prefer it be `ENABLE_UNPREFIXED_BACKDROP_FILTER`